src/api.ts

```diff
@@ -37,7 +43,16 @@ function makeAuthenticatedRequest(
       Authorization: accessToken ? `Bearer ${accessToken}` : undefined,
     },
   };
-  return fetch(getUrl(urlSuffix), options);
+  return fetch(getUrl(urlSuffix), options).then((res) => {
+    if ((options.method ?? "GET") !== "GET") {
+      window.dispatchEvent(
+        new CustomEvent(DATA_MUTATION_EVENT, {
+          detail: { mode: consistencyAction ?? "refetch" },
```

Comment:
**note**: @jossi87 - if we want to change the default (ie, put it back to invalidation), this is where we'd do it.